spec/models/videogame_spec.rb

```diff
@@ -0,0 +1,10 @@
+require 'rails_helper'
+
+RSpec.describe Videogame, type: :model do
+  context 'validation test' do
+    it 'ensures name presence' do
+      game = Videogame.new(name: 'fifa').save
+       expect(game).to eq(true)
+     end
+  end
+end
```

Comment:
Nice! I would also like to test that if a name is not provided that it is not valid to ensure the validation is working correctly